__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../db/app")
+const request = require("supertest")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data/")
+const db = require("../db/connection")
+require("jest-sorted")
+
+beforeEach(() => {
+    return seed(data);
+  });
+  afterAll(() => {
+    return db.end();
+  });
+
+describe("GET /api/topics",()=>{
+    test("responds 200 status and all topics with its data",()=>{
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) =>{
+            expect(body.length).toBe(3)
```

Comment:
Your responses need to be an object with a key you've set - probably a key of `topics` for this one.